### ISSUE TITLE
timeFormat messes up hour between timezones

### ISSUE THREAD

**Issue by paddy01** [bevacqua/rome#192]
I'm using inline calendar and use select able time with 30min intervals.

If I have ' timeFormat: "HH:mm" ' it displays hour wrong but if I change it to ' timeFormat: "HH:mm ZZ" ' that part seems to be working but it becomes rather ugly in the calendar.

Might be that moment has to do with this but I find it strange that a change in timeFormat actually amend the display issue ?
If I do console.log dump from input value and rome.getDateString the hour has been modified from input to output without the "ZZ" addon ?

Just to be sure I actually separated moment and rome and import the latest release of both (using rome.standalone).

**Comment by Tampa** [comment]
You are correct, moment automatically converts to the local timezone of whoever uses rome in their particular browser session. Annoyingly either have to use moment separately to force timezones or convert things. Not sure why rome does not have a way to set timezones when it comes bundled with moment, which can do that easily.
